lib/seam/wait_for_action_attempt.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Seam
+  module Http
+    module WaitForActionAttempt
```

Comment:
These errors are too nested. Thinking about it a bit more, we can lift these out of `Http` module. They are actually not http specific at all, and users will need to handle them in their app, so best if we keep it simple for them.

```
module Seam
	  class ActionAttemptError < StandardError
	  end
	  class ActionAttemptFailedError < StandardError
	  end
	  class ActionAttemptTimeoutError < StandardError
	  end
end
```

Reply:
781f22b36a4386cfa8687f004b7412e9d79641cf